python/PDFs/basic/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ add_library(
   VoigtianPdf.cpp)
 
 target_link_libraries(_Basic PRIVATE _goofit_python)
-set_target_properties(_Basic PROPERTIES FOLDER python/PDFs)
+set_target_properties(_Basic PROPERTIES FOLDER python/PDFs CUDA_RESOLVE_DEVICE_SYMBOLS ON)
```

Comment:
This still bothers me - we shouldn't be forcing the device unconditionally, we need to be able to compile for arbitrary devices (especially such as in our testing CI runs!)

Reply:
this has been added in a PR by Florian: https://github.com/GooFit/GooFit/pull/272. I think it was needed to use python bindings in with CUDA 11. Maybe we can remove this from this commit. @mdsokoloff, do you know if this was needed for your tests?